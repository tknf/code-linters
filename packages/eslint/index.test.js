const ESLint = require("eslint").ESLint;

const { base, react } = require("./index");

describe.concurrent("eslint-config", () => {
  describe("base", () => {
    test("should match snapshot for", async () => {
      const config = new ESLint({
        baseConfig: base,
        overrideConfigFile: true,
      }).calculateConfigForFile("test.ts");
      expect(config).toMatchSnapshot();
    });
  });

  describe("react", () => {
    test("should match snapshot for", async () => {
      const config = new ESLint({
        baseConfig: react,
        overrideConfigFile: true,
      }).calculateConfigForFile("test.tsx");
      expect(config).toMatchSnapshot();
    });
  });
});
