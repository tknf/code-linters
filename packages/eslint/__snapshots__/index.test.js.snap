// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`eslint-config > base > should match snapshot for 1`] = `Promise {}`;

exports[`eslint-config > react > should match snapshot for 1`] = `Promise {}`;
